src/sentry/api/endpoints/group_similar_issues.py

```diff
@@ -19,7 +19,15 @@ def get(self, request, group):
 
         results = filter(
             lambda (group_id, scores): group_id != group.id,
-            features.compare(group, limit=limit)
+            features.compare(
+                group,
+                limit=limit,
+                thresholds={
```

Comment:
Yes, this is a super leaky abstraction right now and I feel terrible about it. :(